feature/notification/src/main/java/com/iteneum/notification/presentation/NotificationListView.kt

```diff
@@ -9,111 +9,113 @@ import androidx.compose.material3.Divider
 import androidx.compose.material3.MaterialTheme
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.font.FontWeight
+import androidx.hilt.navigation.compose.hiltViewModel
 import com.iteneum.designsystem.theme.LeasePertTheme
 import com.iteneum.notification.R
 import com.iteneum.notification.domain.models.NotificationDomain
 
+/**
+ * [NotificationListContent] is a Content that show the list of notifications
+ * [viewModel] this property is one instance of the NotificationViewModel, here is where al the information or actions it will be executed
+ *
+ * @author Daniel Roldan && Usiel Filiberto Garcia Jimenez
+ */
 @Composable
-fun NotificationListView() {
-    NotificationListContent()
+fun NotificationListView(
+    viewModel: NotificationViewModel = hiltViewModel(),
+) {
+    LaunchedEffect(true) {
+        //viewModel.getInformation()
+    }
+    NotificationListContent(
+        listToday = viewModel.dataToday,
+        listWeek = viewModel.dataWeek
+    )
 }
 
 /**
  * [NotificationListContent] is a Content that show the list of notifications
+ * [listToday] is a list what comes form the backend form API and brings the notifications from today
+ * [listWeek] is a list what comes form the backend form API and brings the notifications from this week
  *
- * @author Daniel Roldan
+ * @author Daniel Roldan && Usiel Filiberto Garcia Jimenez
```

Comment:
if you need modified docuemntation you need to add  @modifiedBy Irving Gonzalez

<img width="333" alt="image" src="https://github.com/irv205/droid_leaspert/assets/42255248/70513c18-72e9-42a7-8d97-6dc922808ee2">